ipynb/examples/energy_meter/EnergyMeter_AEP.ipynb

```diff
@@ -0,0 +1,413 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<font size=\"8\">Energy Meter Examples</font>\n",
+    "<br>\n",
+    "<font size=\"5\">ARM Energy Probe</font>\n",
+    "<br>\n",
+    "<hr>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook shows how to use the `AEP` energy meters.\n",
+    "\n",
+    "*NOTE*: `caiman` is required to collect data from the probe. Instructions on how to install it can be found here https://github.com/ARM-software/caiman"
```

Comment:
I would better link:
https://github.com/ARM-software/lisa/wiki/Energy-Meters-Requirements#arm-energy-probe-aep
.. where we have much more info about the installation and usage of the tool.